File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.swift
import Foundation

func solution(_ ineq:String, _ eq:String, _ n:Int, _ m:Int) -> Int {
    if eq == "=" {
        if ineq == ">" {
            return n >= m ? 1 : 0
        } else {
            return n <= m ? 1 : 0
        }
    } else {
        if ineq == ">" {
            return n > m ? 1 : 0
        } else {
            return n < m ? 1 : 0
        }
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.cs
using System;

public class Solution {
    public int solution(string ineq, string eq, int n, int m) {
        string oper = ineq + eq;
            
        if (oper == ">=") return n >= m ? 1 : 0;
        else if (oper == "<=") return n <= m ? 1 : 0;
        else if (oper == ">!") return n > m ? 1 : 0;                    
        else return n < m ? 1 : 0;
    }
}